rust/worker/src/blockstore/arrow/block/delta.rs

```diff
@@ -268,13 +285,23 @@ mod test {
             delta.add(prefix, key.as_str(), value.as_str());
         }
         let size = delta.get_size::<&str, &str>();
-        block_manager.commit::<&str, &str>(&delta);
+        let block = block_manager.commit::<&str, &str>(&delta);
+        let mut values_before_flush = vec![];
+        for i in 0..n {
+            let key = format!("key{}", i);
+            let read = block.get::<&str, &str>("prefix", &key);
+            values_before_flush.push(read.unwrap().to_string());
```

Comment:
nit - would prefer if the test here collected the values we generate instead of reading them from the block. avoid bugs where we persist the value wrong but its consistent